[35mDEBUG   [0m tests.conftest:conftest.py:51 Running fixture setup: test_id
[35mDEBUG   [0m tests.conftest:conftest.py:57 Running test: test_light_push_with_ephemeral with id: 2025-02-03_03-06-04__20891a3b-68e9-4ef1-9731-d2a618d88768
[35mDEBUG   [0m src.steps.common:common.py:18 Running fixture setup: common_setup
[35mDEBUG   [0m src.steps.light_push:light_push.py:28 Running fixture setup: light_push_setup
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:19 Docker client initialized with image wakuorg/nwaku:latest
[35mDEBUG   [0m src.node.waku_node:waku_node.py:85 WakuNode instance initialized with log path ./log/docker/receiving_node1_2025-02-03_03-06-04__20891a3b-68e9-4ef1-9731-d2a618d88768__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:89 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:22 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:25 Network waku already exists
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:108 Generated random external IP 172.18.26.185
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:101 Generated ports ['64396', '64397', '64398', '64399', '64400']
[35mDEBUG   [0m src.node.waku_node:waku_node.py:439 RLN credentials were not set
[32mINFO    [0m src.node.waku_node:waku_node.py:168 RLN credentials not set or credential store not available, starting without RLN
[35mDEBUG   [0m src.node.waku_node:waku_node.py:170 Using volumes []
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:49 docker run -i -t -p 64396:64396 -p 64397:64397 -p 64398:64398 -p 64399:64399 -p 64400:64400 wakuorg/nwaku:latest --listen-address=0.0.0.0 --rest=true --rest-admin=true --websocket-support=true --log-level=TRACE --rest-relay-cache-capacity=100 --websocket-port=64398 --rest-port=64396 --tcp-port=64397 --discv5-udp-port=64399 --rest-address=0.0.0.0 --nat=extip:172.18.26.185 --peer-exchange=true --discv5-discovery=true --cluster-id=3 --nodekey=cbaaf1abb527496d29b1ae1edb4dfb3ad6ff9cf6dcef7aef6c33d47c6dbbd0e6 --shard=0 --metrics-server=true --metrics-server-address=0.0.0.0 --metrics-server-port=64400 --metrics-logging=true --lightpush=true --relay=true
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:55 docker network connect --ip 172.18.26.185 waku 05c24b7d6008b3b44979311da13d25f48776a4b504bf206cb194a543933cd713
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:58 Container started with ID 05c24b7d6008. Setting up logs at ./log/docker/receiving_node1_2025-02-03_03-06-04__20891a3b-68e9-4ef1-9731-d2a618d88768__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:182 Started container from image wakuorg/nwaku:latest. REST: 64396
[35mDEBUG   [0m src.libs.common:common.py:47 Sleeping for 1 seconds
[1m[31mERROR   [0m src.node.docker_mananger:docker_mananger.py:89 Max retries reached for container 3b6eb3bd87d7. Exiting log stream.
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:64396/health" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"nodeHealth":"Ready","protocolsHealth":[]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:282 Node protocols are initialized !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:64396/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"listenAddresses":["/ip4/172.18.26.185/tcp/64397/p2p/16Uiu2HAm9C7dPxs8izi4WmBk8dWuGdLUZva7R6vc4jXLTrBLakND","/ip4/172.18.26.185/tcp/64398/ws/p2p/16Uiu2HAm9C7dPxs8izi4WmBk8dWuGdLUZva7R6vc4jXLTrBLakND"],"enrUri":"enr:-L24QDh2Z4aug4feTDUyqtQzraBoeC0LXrg94e-ZvBmTpEnLMeBPd8nGxUjsIT06x6OqvAM6wWZENhl-EQ0PxPRDaUECgmlkgnY0gmlwhKwSGrmKbXVsdGlhZGRyc5YACASsEhq5BvuNAAoErBIauQb7jt0DgnJzhQADAQAAiXNlY3AyNTZrMaECzIp8flMEysNhvLwU55eAdlsQuHOH4ljUmAQIfdyzO7KDdGNwgvuNg3VkcIL7j4V3YWt1Mgk"}'
[32mINFO    [0m src.node.waku_node:waku_node.py:287 REST service is ready !!
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:19 Docker client initialized with image wakuorg/nwaku:latest
[35mDEBUG   [0m src.node.waku_node:waku_node.py:85 WakuNode instance initialized with log path ./log/docker/receiving_node2_2025-02-03_03-06-04__20891a3b-68e9-4ef1-9731-d2a618d88768__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:89 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:22 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:25 Network waku already exists
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:108 Generated random external IP 172.18.11.161
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:101 Generated ports ['48123', '48124', '48125', '48126', '48127']
[35mDEBUG   [0m src.node.waku_node:waku_node.py:439 RLN credentials were not set
[32mINFO    [0m src.node.waku_node:waku_node.py:168 RLN credentials not set or credential store not available, starting without RLN
[35mDEBUG   [0m src.node.waku_node:waku_node.py:170 Using volumes []
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:49 docker run -i -t -p 48123:48123 -p 48124:48124 -p 48125:48125 -p 48126:48126 -p 48127:48127 wakuorg/nwaku:latest --listen-address=0.0.0.0 --rest=true --rest-admin=true --websocket-support=true --log-level=TRACE --rest-relay-cache-capacity=100 --websocket-port=48125 --rest-port=48123 --tcp-port=48124 --discv5-udp-port=48126 --rest-address=0.0.0.0 --nat=extip:172.18.11.161 --peer-exchange=true --discv5-discovery=true --cluster-id=3 --nodekey=8cdc21bfb5387c2eab85f5dcdb7b0a43573fdfa41c0691cec5d9b9e6e5ba653b --shard=0 --metrics-server=true --metrics-server-address=0.0.0.0 --metrics-server-port=48127 --metrics-logging=true --lightpush=false --relay=true
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:55 docker network connect --ip 172.18.11.161 waku 81b3810170a5599e2afd54a2d6be9c468a50e654f327b111c25b002e3dc2e4a8
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:58 Container started with ID 81b3810170a5. Setting up logs at ./log/docker/receiving_node2_2025-02-03_03-06-04__20891a3b-68e9-4ef1-9731-d2a618d88768__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:182 Started container from image wakuorg/nwaku:latest. REST: 48123
[35mDEBUG   [0m src.libs.common:common.py:47 Sleeping for 1 seconds
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:48123/health" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"nodeHealth":"Ready","protocolsHealth":[]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:282 Node protocols are initialized !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:48123/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"listenAddresses":["/ip4/172.18.11.161/tcp/48124/p2p/16Uiu2HAmDBVw3J7K4kphRoUjE3r6TFUs2Af2aGpavdzDnsfmhmqm","/ip4/172.18.11.161/tcp/48125/ws/p2p/16Uiu2HAmDBVw3J7K4kphRoUjE3r6TFUs2Af2aGpavdzDnsfmhmqm"],"enrUri":"enr:-L24QBzC3UFeWGkwT2TzlwijB_qY4UYLQ1SFPig9VkaxJqcfKoHTsPnsjRXPeT4dfYa3meLnGbkfT0sz6AhjeCbTKgMCgmlkgnY0gmlwhKwSC6GKbXVsdGlhZGRyc5YACASsEguhBrv8AAoErBILoQa7_d0DgnJzhQADAQAAiXNlY3AyNTZrMaEDB9DfZUkD4uUcbYgI9UNXGzg7O2Clu2LDReiNTJI6RvyDdGNwgrv8g3VkcIK7_oV3YWt1MgE"}'
[32mINFO    [0m src.node.waku_node:waku_node.py:287 REST service is ready !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:48123/admin/v1/peers" -H "Content-Type: application/json" -d '["/ip4/172.18.26.185/tcp/64397/p2p/16Uiu2HAm9C7dPxs8izi4WmBk8dWuGdLUZva7R6vc4jXLTrBLakND"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:19 Docker client initialized with image wakuorg/go-waku:latest
[35mDEBUG   [0m src.node.waku_node:waku_node.py:85 WakuNode instance initialized with log path ./log/docker/lightpush_node1_2025-02-03_03-06-04__20891a3b-68e9-4ef1-9731-d2a618d88768__wakuorg_go-waku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:89 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:22 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:25 Network waku already exists
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:108 Generated random external IP 172.18.146.77
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:101 Generated ports ['9442', '9443', '9444', '9445', '9446']
[35mDEBUG   [0m src.node.waku_node:waku_node.py:439 RLN credentials were not set
[32mINFO    [0m src.node.waku_node:waku_node.py:168 RLN credentials not set or credential store not available, starting without RLN
[35mDEBUG   [0m src.node.waku_node:waku_node.py:170 Using volumes []
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:49 docker run -i -t -p 9442:9442 -p 9443:9443 -p 9444:9444 -p 9445:9445 -p 9446:9446 wakuorg/go-waku:latest --listen-address=0.0.0.0 --rest=true --rest-admin=true --websocket-support=true --log-level=DEBUG --rest-relay-cache-capacity=100 --websocket-port=9444 --rest-port=9442 --tcp-port=9443 --discv5-udp-port=9445 --rest-address=0.0.0.0 --nat=extip:172.18.146.77 --peer-exchange=true --discv5-discovery=true --cluster-id=3 --nodekey=07d4ae3ecf69d2e0c5ce5eee9ac4c5a0b42bb7fad044dcd7e2dd28f4f554b7c0 --min-relay-peers-to-publish=1 --rest-filter-cache-capacity=50 --peer-store-capacity=10 --discv5-bootstrap-node=enr:-L24QDh2Z4aug4feTDUyqtQzraBoeC0LXrg94e-ZvBmTpEnLMeBPd8nGxUjsIT06x6OqvAM6wWZENhl-EQ0PxPRDaUECgmlkgnY0gmlwhKwSGrmKbXVsdGlhZGRyc5YACASsEhq5BvuNAAoErBIauQb7jt0DgnJzhQADAQAAiXNlY3AyNTZrMaECzIp8flMEysNhvLwU55eAdlsQuHOH4ljUmAQIfdyzO7KDdGNwgvuNg3VkcIL7j4V3YWt1Mgk --lightpushnode=/ip4/172.18.26.185/tcp/64397/p2p/16Uiu2HAm9C7dPxs8izi4WmBk8dWuGdLUZva7R6vc4jXLTrBLakND --lightpush=true --relay=false --pubsub-topic=/waku/2/rs/3/1
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:55 docker network connect --ip 172.18.146.77 waku a19f4874489b5da10a3e4c61a8c757b47e0b015d625e2c8b7149f3297b1ba7fc
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:58 Container started with ID a19f4874489b. Setting up logs at ./log/docker/lightpush_node1_2025-02-03_03-06-04__20891a3b-68e9-4ef1-9731-d2a618d88768__wakuorg_go-waku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:182 Started container from image wakuorg/go-waku:latest. REST: 9442
[35mDEBUG   [0m src.libs.common:common.py:47 Sleeping for 1 seconds
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:9442/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"enrUri":"enr:-KG4QJF-RHjW9SrHU0MCxEid6v4NSWdG2DMr_eT0JS_MYq61JvxCmAD7AHEuWhRtJKHmTdU2ewcClO_qKm9r9knTSoqGAZTJxb9RgmlkgnY0gmlwhKwRAASCcnOFAAMBAAGJc2VjcDI1NmsxoQNa8sN08qIvv0KMXWuSOxvvLwzkGhTg3em4zr11iXqd8YN0Y3CCJOODdWRwgiTlhXdha3UyCA","listenAddresses":["/ip4/127.0.0.1/tcp/9443/p2p/16Uiu2HAmJn1k9RTmxZzPkJqtipbA5ETyri7isx5b5dCppQ6Ep5uJ","/ip4/127.0.0.1/tcp/9444/ws/p2p/16Uiu2HAmJn1k9RTmxZzPkJqtipbA5ETyri7isx5b5dCppQ6Ep5uJ","/ip4/172.17.0.4/tcp/9443/p2p/16Uiu2HAmJn1k9RTmxZzPkJqtipbA5ETyri7isx5b5dCppQ6Ep5uJ","/ip4/172.17.0.4/tcp/9444/ws/p2p/16Uiu2HAmJn1k9RTmxZzPkJqtipbA5ETyri7isx5b5dCppQ6Ep5uJ"]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:287 REST service is ready !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:9442/admin/v1/peers" -H "Content-Type: application/json" -d '{"multiaddr": "/ip4/172.18.26.185/tcp/64397/p2p/16Uiu2HAm9C7dPxs8izi4WmBk8dWuGdLUZva7R6vc4jXLTrBLakND", "protocols": ["/vac/waku/relay/2.0.0"], "shards": [0, 1, 2, 3, 4, 5, 6, 7, 8]}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'null'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:9442/admin/v1/peers" -H "Content-Type: application/json" -d '{"multiaddr": "/ip4/172.18.11.161/tcp/48124/p2p/16Uiu2HAmDBVw3J7K4kphRoUjE3r6TFUs2Af2aGpavdzDnsfmhmqm", "protocols": ["/vac/waku/relay/2.0.0"], "shards": [0, 1, 2, 3, 4, 5, 6, 7, 8]}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'null'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:64396/relay/v1/subscriptions" -H "Content-Type: application/json" -d '["/waku/2/rs/3/0"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:48123/relay/v1/subscriptions" -H "Content-Type: application/json" -d '["/waku/2/rs/3/0"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m tests.light_push.test_publish:test_publish.py:214 Running test with Ephemeral True
[35mDEBUG   [0m src.steps.light_push:light_push.py:121 Lightpushing message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:9442/lightpush/v1/message" -H "Content-Type: application/json" -d '{"pubsubTopic": "/waku/2/rs/3/0", "message": {"payload": "TGlnaHQgcHVzaCB3b3JrcyEh", "contentTopic": "/myapp/1/latest/proto", "timestamp": 1738551968612804096, "ephemeral": true}}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'true'
[35mDEBUG   [0m src.libs.common:common.py:47 Sleeping for 0.1 seconds
[35mDEBUG   [0m src.steps.light_push:light_push.py:125 Checking that peer NODE_1:wakuorg/nwaku:latest can find the lightpushed message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:64396/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F0" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'[{"payload":"TGlnaHQgcHVzaCB3b3JrcyEh","contentTopic":"/myapp/1/latest/proto","version":0,"timestamp":1738551968612804096,"ephemeral":true}]'
[35mDEBUG   [0m src.steps.light_push:light_push.py:125 Checking that peer NODE_2:wakuorg/nwaku:latest can find the lightpushed message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:48123/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F0" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'[{"payload":"TGlnaHQgcHVzaCB3b3JrcyEh","contentTopic":"/myapp/1/latest/proto","version":0,"timestamp":1738551968612804096,"ephemeral":true}]'
[35mDEBUG   [0m tests.light_push.test_publish:test_publish.py:214 Running test with Ephemeral False
[35mDEBUG   [0m src.steps.light_push:light_push.py:121 Lightpushing message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:9442/lightpush/v1/message" -H "Content-Type: application/json" -d '{"pubsubTopic": "/waku/2/rs/3/0", "message": {"payload": "TGlnaHQgcHVzaCB3b3JrcyEh", "contentTopic": "/myapp/1/latest/proto", "timestamp": 1738551968730432512, "ephemeral": false}}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'true'
[35mDEBUG   [0m src.libs.common:common.py:47 Sleeping for 0.1 seconds
[35mDEBUG   [0m src.steps.light_push:light_push.py:125 Checking that peer NODE_1:wakuorg/nwaku:latest can find the lightpushed message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:64396/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F0" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'[{"payload":"TGlnaHQgcHVzaCB3b3JrcyEh","contentTopic":"/myapp/1/latest/proto","version":0,"timestamp":1738551968730432512,"ephemeral":false}]'
[35mDEBUG   [0m src.steps.light_push:light_push.py:125 Checking that peer NODE_2:wakuorg/nwaku:latest can find the lightpushed message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:48123/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F0" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'[{"payload":"TGlnaHQgcHVzaCB3b3JrcyEh","contentTopic":"/myapp/1/latest/proto","version":0,"timestamp":1738551968730432512,"ephemeral":false}]'
[35mDEBUG   [0m tests.conftest:conftest.py:59 Running fixture teardown: test_setup
[35mDEBUG   [0m tests.conftest:conftest.py:83 Running fixture teardown: close_open_nodes
[35mDEBUG   [0m src.node.waku_node:waku_node.py:226 Stopping container with id 05c24b7d6008
[35mDEBUG   [0m src.node.waku_node:waku_node.py:233 Container stopped.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:226 Stopping container with id 81b3810170a5
[1m[31mERROR   [0m src.node.docker_mananger:docker_mananger.py:89 Max retries reached for container 05c24b7d6008. Exiting log stream.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:233 Container stopped.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:226 Stopping container with id a19f4874489b
[35mDEBUG   [0m src.node.waku_node:waku_node.py:233 Container stopped.
[35mDEBUG   [0m tests.conftest:conftest.py:98 Running fixture teardown: check_waku_log_errors
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:144 No errors found in the waku logs.
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:144 No errors found in the waku logs.
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:144 No errors found in the waku logs.